packages/tempus/src/Tempus_StepperIMEX_RK_Partition_impl.hpp

```diff
@@ -679,11 +674,78 @@ getDefaultStepperState()
 template<class Scalar>
 void StepperIMEX_RK_Partition<Scalar>::describe(
    Teuchos::FancyOStream               &out,
-   const Teuchos::EVerbosityLevel      /* verbLevel */) const
+   const Teuchos::EVerbosityLevel      verbLevel) const
+{
+  out << std::endl;
+  Stepper<Scalar>::describe(out, verbLevel);
+  StepperImplicit<Scalar>::describe(out, verbLevel);
+
+  out << "--- StepperIMEX_RK_Partition ---\n";
+  out << "  explicitTableau_   = " << explicitTableau_ << std::endl;
+  out << "  implicitTableau_   = " << explicitTableau_ << std::endl;
+  out << "  xTilde_            = " << xTilde_  << std::endl;
+  out << "  stageZ_            = " << stageZ_  << std::endl;
+  out << "  stageF_.size()     = " << stageF_.size() << std::endl;
+  int numStages = stageF_.size();
+  for (int i=0; i<numStages; ++i)
+    out << "    stageF_["<<i<<"] = " << stageF_[i] << std::endl;
+  out << "  stageGx_.size()    = " << stageGx_.size() << std::endl;
+  numStages = stageGx_.size();
+  for (int i=0; i<numStages; ++i)
+    out << "    stageGx_["<<i<<"] = " << stageGx_[i] << std::endl;
+  out << "  stepperRKObserver_ = " << stepperRKObserver_ << std::endl;
+  out << "  order_             = " << order_ << std::endl;
+  out << "--------------------------------" << std::endl;
```

Comment:
Should this be guarded for `VERB_EXTREME`?

Reply:
I am not a big fan of Teuchos verbosity, and have leaned towards Roger's approach of having flags for various outputs.  In this case, the describe() function is currently only called when there is a failed initialization and information on the stepper's state is needed on what might not be initialized.

Reply:
OK